ntbs-service/Pages/Notifications/Edit/_ClinicalDetailsDates.cshtml

```diff
@@ -9,7 +9,7 @@
 
 <h2>Clinical Dates</h2>
 
-<date-comparison inline-template number-of-dates="6">
+<date-comparison inline-template v-bind:number-of-dates="6">
```

Comment:
is this number actually used anyhwere?

Reply:
It's used to clear the "hidden" class on the date warnings, using this prop so that all the date warning elements are hit not just ones that have dates present in them. Lines 19-21 of DateComparison.ts